master/internal/api_experiment.go

```diff
@@ -419,32 +420,38 @@ func (a *apiServer) DeleteExperiments(
 		return nil, status.Errorf(codes.Internal, "failed to get the user: %s", err)
 	}
 
-	results, experiments, err := experiment.DeleteExperiments(ctx, req.ExperimentIds, req.Filters)
+	results, experiments, err := experiment.DeleteExperiments(ctx, req.ExperimentIds, req.Filters, req.SearchFilter)
 	if err != nil {
 		return nil, err
 	}
 
+	// gather resolved ids
+	var expIDs []int32
+	for _, exp := range experiments {
+		expIDs = append(expIDs, int32(exp.ID))
+	}
```

Comment:
question (non-blocking): Why is this cast necessary now, when it had been done prior?